modules/charon-core/src/main/java/org/wso2/charon3/core/utils/codeutils/FilterTreeManager.java

```diff
@@ -43,7 +43,7 @@
 public class FilterTreeManager {
 
     private StreamTokenizer input;
-    protected List<String> tokenList = null;
+    protected List<String> newTokenList = null;
```

Comment:
having this variable name as `newTokenList` does not make any sense to me. How about keeping this as it is and creating a new variable inside the constructor with name `newTokenList`

Reply:
Fixed with 1765dd487f049e67b7e5009abc8bda6825cae68a